app/api/resources/users.py

```diff
@@ -5,6 +5,7 @@
 from app.api.ms_api_utils import post_request, BASE_MS_API_URL
 from app import messages
 from app.api.models.user import *
```

Comment:
I would prefer importing explicitly rather than *
https://medium.com/@s16h/importing-star-in-python-88fe9e8bd4d2

Reply:
Yeah, I actually confused to what I actually have to import. 
Tried to use the function name as import but the tests failed because of that.
<img width="992" alt="Screen Shot 2020-06-22 at 12 53 41 am" src="https://user-images.githubusercontent.com/29667122/85227840-2670c080-b423-11ea-9c13-a0ca6041f19f.png">

But if I use * the tests are discovered and passed successfully
<img width="983" alt="Screen Shot 2020-06-22 at 12 54 06 am" src="https://user-images.githubusercontent.com/29667122/85227858-3ee0db00-b423-11ea-9dfc-8b7c9e5af6b7.png">

And I only have that one function on the file to be imported
<img width="734" alt="Screen Shot 2020-06-22 at 12 54 16 am" src="https://user-images.githubusercontent.com/29667122/85227871-4dc78d80-b423-11ea-88d1-00a849d7fd36.png">

Can you tell me what exactly went wrong here?

Reply:
I think there might be possibly clashing names within the namespaces. For now, it is okay to use *

Reply:
Great 👍